tests/meta-mender-ci/recipes-mender/mender-monitor/mender-monitor_%.bbappend

```diff
@@ -0,0 +1,16 @@
+# Produce custom level logs when MENDER_CI_LOGLEVEL is enabled
+do_install_append_class-target_mender-image_mender-systemd() {
+    if ${@'true' if d.getVar('MENDER_CI_LOGLEVEL') else 'false'}; then
+        local logging_level=""
+        if [ "${MENDER_CI_LOGLEVEL}" = "debug" ]; then
+            logging_level="DEBUG"
+        fi
+
+        if [ -n "$logging_level" ]; then
+            sed -i "s|CONFIG_LOG_LEVEL=.*|CONFIG_LOG_LEVEL=${logging_level}|" \
```

Comment:
nit pick, ignore if you feel like it:
* what if for some reason config/config.sh does not contain `CONFIG_LOG_LEVEL`?
* what if it contains: `# CONFIG_LOG_LEVEL = `?

Reply:
Only to be used for local debugging, so it is not worth worrying too much. See also how is done for `mender` and `mender-connect`.